internal/builtinfunctions/functions.go

```diff
@@ -231,21 +231,23 @@ func getStringToBoolFunction() schema.CallableFunction {
 			schema.NewStringSchema(
 				nil,
 				nil,
-				regexp.MustCompile(`[Tt]rue|TRUE|[Ff]alse|FALSE|[tTfF]|[01]$`),
+				regexp.MustCompile(`(?i)^(?:true|false|[tf]|[01])$`),
 			)},
 		schema.NewBoolSchema(),
 		true,
 		schema.NewDisplayValue(
 			schema.PointerTo("stringToBool"),
 			schema.PointerTo(
 				"Interprets the input as a boolean.\n"+
```

Comment:
Again, consistency in phrasing is good. If we're using "Converts the" for other conversion functions, then using "Interprets the" here breaks the pattern and obscures the meaning.

Maybe even "Converts a case insensitive string representation of a boolean value to a boolean type"??

Reply:
That seems less clear to me.